src/main/java/core/basesyntax/dao/ma/MentorDaoImpl.java

```diff
@@ -2,6 +2,7 @@
 
 import core.basesyntax.model.ma.Mentor;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 
 public class MentorDaoImpl extends PersonDaoImpl implements MentorDao {
```

Comment:
how about refactoring save() method in PersonDaoImpl using transaction?

Reply:
Sure. Done